package.json

```diff
@@ -23,6 +23,8 @@
   },
   "homepage": "https://github.com/andela/freyja-ah-frontend#readme",
   "dependencies": {
+    "@babel/polyfill": "^7.4.4",
+    "@babel/runtime": "^7.5.4",
```

Comment:
@Ochowo we @babel/polyfill and @babel/runtime is redundant, we have handled that by adding @babel/plugin-transform-runtime to our .babelrc

Reply:
I already took them out but it came back after I rebased. I will take them out and push. thank you